SU2_CFD/src/fluid/CMutationTCLib.cpp

```diff
@@ -27,46 +27,201 @@
 
 #include "../../include/fluid/CMutationTCLib.hpp"
 
+//#if defined(HAVE_MPP) && !defined(CODI_REVERSE_TYPE) && !defined(CODI_FORWARD_TYPE)
+
 CMutationTCLib::CMutationTCLib(const CConfig* config, unsigned short val_nDim): CNEMOGas(config, val_nDim){
```

Comment:
And this entire file as well (forget my earlier comment about the constructor because you would have to apply it to all the methods that reference the member "mix")